yarn/src/main/scala/org/apache/spark/deploy/yarn/AMDelegationTokenRenewer.scala

```diff
@@ -60,8 +60,11 @@ private[yarn] class AMDelegationTokenRenewer(
 
   private val hadoopUtil = YarnSparkHadoopUtil.get
 
-  private val daysToKeepFiles = sparkConf.getInt("spark.yarn.credentials.file.retention.days", 5)
-  private val numFilesToKeep = sparkConf.getInt("spark.yarn.credentials.file.retention.count", 5)
+  private val credentialsFile = sparkConf.get("spark.internal.yarn.credentials.file")
+  private val daysToKeepFiles =
+    sparkConf.getInt("spark.internal.yarn.credentials.file.retention.days", 5)
```

Comment:
These two should not be marked internal - these are undocumented for now, but these can be made public at some point.